tests/std/tests/P1502R1_standard_library_header_units/test.cpp

```diff
@@ -0,0 +1,973 @@
+        const string_view sv{buf, size(buf)};
+        assert(sv == "3.14");
+    }
+
+    {
+        puts("Testing <chrono>.");
+#ifdef MSVC_INTERNAL_TESTING // TRANSITION, DevCom-1159995 (UDLs)
+        constexpr chrono::seconds dur = 3min;
+#else // ^^^ no workaround / workaround vvv
+        constexpr chrono::seconds dur = chrono::minutes{3};
+#endif // ^^^ workaround ^^^
+        assert(dur.count() == 180);
+        static_assert(dur.count() == 180);
+    }
+
+    {
+        puts("Testing <codecvt>.");
+#if 0 // TRANSITION, DevCom-1161187 (access control), VSO-1236034 (error LNK2005: _Yarn::operator= already defined)
+        const string utf8_koshka_cat{"\xD0\xBA\xD0\xBE\xD1\x88\xD0\xBA\xD0\xB0_\xF0\x9F\x90\x88"};
+        const wstring utf16_koshka_cat{L"\x043A\x043E\x0448\x043A\x0430_\xD83D\xDC08"};
+        wstring_convert<codecvt_utf8_utf16<wchar_t>> conv;
+        assert(conv.from_bytes(utf8_koshka_cat) == utf16_koshka_cat);
+        assert(conv.to_bytes(utf16_koshka_cat) == utf8_koshka_cat);
+#endif // ^^^ no workaround ^^^
+
+        static_assert(static_cast<int>(codecvt_mode::consume_header) == 4); // TRANSITION, DevCom-1160041 (deprecated)
+    }
+
+    {
+        puts("Testing <compare>.");
+        assert(is_lt(10 <=> 20));
+        static_assert(is_lt(10 <=> 20));
+    }
+
+    {
+        puts("Testing <complex>.");
+        constexpr complex<double> c{3.0, 4.0};
+        assert(norm(c) == 25.0);
+        static_assert(norm(c) == 25.0);
+    }
+
+    {
+        puts("Testing <concepts>.");
+        static_assert(signed_integral<short>);
+        static_assert(!signed_integral<unsigned short>);
+        static_assert(!signed_integral<double>);
+    }
+
+    {
+        puts("Testing <condition_variable>.");
+        condition_variable cv;
+        mutex mut;
+        vector<int> vec = {5};
+
+        thread odd{[&cv, &mut, &vec] {
+            unique_lock<mutex> lk{mut};
+
+            while (vec.size() < 6) {
+                cv.wait(lk, [&vec] { return vec.size() % 2 == 1; });
+                const int n = vec.back();
+                vec.push_back(n * 10 + 1);
+                cv.notify_one();
+            }
+        }};
+
+        thread even{[&cv, &mut, &vec] {
+            unique_lock<mutex> lk{mut};
+
+            while (vec.size() < 7) {
+                cv.wait(lk, [&vec] { return vec.size() % 2 == 0; });
+                const int n = vec.back();
+                vec.push_back(n * 10 + 2);
+                cv.notify_one();
+            }
+        }};
+
+        odd.join();
+        even.join();
+
+        const vector<int> expected = {5, 51, 512, 5121, 51212, 512121, 5121212};
+        assert(vec == expected);
+
+        static_assert(static_cast<int>(cv_status::no_timeout) == 0);
+        static_assert(static_cast<int>(cv_status::timeout) == 1);
+    }
+
+    {
+        puts("Testing <coroutine>.");
+        const coroutine_handle<> handle{};
+        assert(handle.address() == nullptr);

```

Comment:
```suggestion
        constexpr coroutine_handle<> handle{};
        assert(handle.address() == nullptr);
        static_assert(handle.address() == nullptr);
```

Reply:
Done. I think I was concerned whether `coroutine_handle<>` is a literal type (which doesn't seem to be mentioned in the Standard) but this does compile and pass.